examples/internal-lb-cloud-run/main.tf

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_compute_network" "default" {
+  name                    = "lb-internal-cloud-run"
+  auto_create_subnetworks = "false"
+  project                 = var.project_id
+}
+
+resource "google_compute_subnetwork" "default" {
+  name          = "lb-internal-subnet-cloud-run"
+  ip_cidr_range = "10.1.2.0/24"
+  network       = google_compute_network.default.id
+  region        = var.region
+  project       = var.project_id
+}
+
+resource "google_compute_subnetwork" "proxy_only" {
+  name          = "proxy-only-subnet-int"
+  ip_cidr_range = "10.129.0.0/23"
+  network       = google_compute_network.default.id
+  purpose       = "REGIONAL_MANAGED_PROXY"
+  region        = var.region
+  project       = var.project_id
+  role          = "ACTIVE"
+}
+
+module "backend-service" {
+  source                        = "GoogleCloudPlatform/cloud-run/google//modules/v2"
+  version                       = "~> 0.16.3"
+  project_id                    = var.project_id
+  location                      = var.region
+  service_name                  = "bs-2002"
+  containers                    = [{ "container_name" = "", "container_image" = "gcr.io/cloudrun/hello" }]
+  service_account_project_roles = ["roles/run.invoker"]
+  members                       = ["allUsers"]
```

Comment:
Let's remove this (here and throughout)

Reply:
`allUsers` are needed to allow access to the cloud-run service from internal LB.